qa/suites/upgrade/telemetry-upgrade/reef-x/1-tasks.yaml

```diff
@@ -1,34 +1,51 @@
+overrides:
+  ceph:
+    log-ignorelist:
+      - mons down
+      - mon down
+      - MON_DOWN
+      - out of quorum
+      - PG_
+      - OSD_DOWN
+      - POOL_APP_NOT_ENABLED
+      - pgs degraded
+      - pg degraded
+      - object degraded
+      - objects degraded
+      - FS_DEGRADED
+      - MDS_ALL_DOWN
+      - OSD_UPGRADE_FINISHED
+      - do not have an application enabled
+      - is down
 tasks:
 - install:
-    branch: pacific
+    branch: reef
     exclude_packages:
       - ceph-volume
 - print: "**** done install task..."
-- print: "**** done start installing pacific cephadm ..."
+- print: "**** done start installing reef cephadm ..."
 - cephadm:
-    image: quay.io/ceph/daemon-base:latest-pacific
```

Comment:
Why are we using [quay.ceph.io/ceph-ci/ceph:reef](https://quay.ceph.io/ceph-ci/ceph:reef) instead of [quay.io/ceph/daemon-base:latest-reef](https://quay.io/ceph/daemon-base:latest-reef)?

Reply:
quay.io/ceph/daemon-base:latest-reef points to the latest reef tag (v18.2.2), not the tip of reef, which is actually what we want to test.

Pulling the daemon-base:latest-reef image shows it is running v18.2.2:
```
lflores:ceph$ podman run --rm -it daemon-base:latest-reef
[root@d4d9a0eb04be /]# ceph -v
ceph version 18.2.2 (531c0d11a1c5d39fbfe6aa8a521f023abf3bf3e2) reef (stable)
```

But it is important that we test the tip of reef in case there is a fix there that we want to pick up, such as the k8sevents backport.

Also, I referenced this commit to make the change: https://github.com/ceph/ceph/commit/d0381ed7e68a6de7c9ff1e4220698ed8eab28ac6